### ISSUE TITLE
[Bug]: Unable to detect Brave on macOS

### ISSUE THREAD

**Issue by srirambv** [httptoolkit/httptoolkit#394]
### Repro steps

1. Install HTTPToolkit
2. Install Brave Beta and Brave Nightly
3. Launch HTTPToolkit
4. Brave isn't recognized 

### How often does this bug happen?

Every time

### The desktop OS you're using

macOS Version 13.2.1 (Build 22D68)

### Details of other apps/devices

Both Brave Beta and Nightly are clean installs

### Error screenshot

![image](https://user-images.githubusercontent.com/17010094/226306330-3913c5d8-8571-4528-af73-3e23ec64a984.png)

### Any other info?

Tried changing the path in `browsers.json` to point to the actual profile path and kept only one entry for Brave browser but still doesn't detect the browser(s).

**Comment by pimterry** [comment]
Ah, interesting! Yes I think only the stable channel is detected, not beta/canary/etc. If you're interested in this, do you want to open a PR for https://github.com/httptoolkit/browser-launcher? Once it's detected & supported there, it'll be a 1 minute job to make it work in HTTP Toolkit itself.

**Comment by srirambv** [comment]
> Yes I think only the stable channel is detected, not beta/canary/etc

Yes its weird coz on Windows and Linux I was able to access both Beta and Nightly granted it would detect Beta if both Beta and Nightly are installed. 

>  If you're interested in this, do you want to open a PR for https://github.com/httptoolkit/browser-launcher?

Will do 👍
